### PR TITLE
gui: Add option to allow disable display compile shaders dialog.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About
Request by lot user
- Add option in setting dialog for can disable display dialog of compile ;shaders

### CODE REVIEW

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -741,6 +741,10 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::Checkbox("Texture Cache", &emuenv.cfg.texture_cache);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Uncheck the box to disable texture cache.");
+        ImGui::SameLine();
+        ImGui::Checkbox("Show Compile Shaders", &emuenv.cfg.show_compile_shaders);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to disable display of compile shaders dialog.");
```
Comment (Macdu) [inline vita3k/gui/src/settings_dialog.cpp:747]:
```suggestion
            ImGui::SetTooltip("Uncheck the box to disable the display of compile shaders dialogs.");
```
I'm not entirely sure about this.

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:747]: is wrong, is only one dialog ^^

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp]: but i have pushed fix typo for the missing

Comment (Macdu) [reply vita3k/gui/src/settings_dialog.cpp]: It's one dialog which shows up multiple time so both are right I guess. If it's only one dialog it should be "display of the compile shader dialog."  I think.

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp]: is because it show count of shaders :) so shaers but dialog
one dialog for few shaders :)

Comment (Macdu) [reply vita3k/gui/src/settings_dialog.cpp]: Ah sorry, that was a typo, I did not meant to tell you to replace shaders by shader.

**Review by Macdu — APPROVED** [review]

File: vita3k/gui/src/settings_dialog.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp]: no problem :)

**Pull request merged**